addons/mail/views/mail_mail_views.xml

```diff
@@ -66,7 +66,7 @@
             <field name="name">mail.mail.tree</field>
             <field name="model">mail.mail</field>
             <field name="arch" type="xml">
-                <tree string="Emails" decoration-muted="state in ('sent', 'cancel')" decoration-info="state=='outgoing'" decoration-danger="state=='exception'">
```

Comment:
As your commits seem to contain a little bit of everything (perf + view), maybe we will use a merge commit?
So we should update the PR description to explain the global purpose and point to the commit messages for details.